docker-compose.yml

```diff
@@ -14,30 +14,28 @@ services:
 
   web:
     extends: base
-    command: web -b 0.0.0.0:8000
+    command: web -b 0.0.0.0:8000 --reload
     ports:
       - "8080:8000"
-    links:
-      - mysql
-      - redis
     environment:
-      DEBUG: 1
+      - DEBUG=1
+      - ALLOWED_HOSTS=127.0.0.1,localhost,web
 
   worker:
     extends: base
-    command: worker -l info --queues localhost,celery
-    links:
-      - prometheus
-      - mysql
-      - redis
+    command: worker -l info --queues localhost,celery,prometheus,alertmanager
     volumes:
       - ./docker:/etc/prometheus
 
   prometheus:
     image: prom/prometheus
-    links:
-      - alertmanager
-      - blackbox
+    user: "root"
+    command:
+      - --config.file=/etc/prometheus/prometheus.yml
+      - --storage.tsdb.path=/prometheus
+      - --web.console.libraries=/usr/share/prometheus/console_libraries
+      - --web.console.templates=/usr/share/prometheus/consoles
+      - --web.enable-lifecycle
```

Comment:
I think all the defaults *should* be ok, so I'm unsure the reason to add them here.

Reply:
The command option is overridden to add the `--web.enable-lifecycle` which is required to allow prometheus to reload its configuration. 

```
Prometheus can reload its configuration at runtime. If the new configuration is not well-formed, the changes will not be applied. A configuration reload is triggered by sending a SIGHUP to the Prometheus process or sending a HTTP POST request to the /-/reload endpoint (when the --web.enable-lifecycle flag is enabled). This will also reload any configured rule files
```

Reply:
That's a good catch then. I suppose I need to test more carefully then because I never noticed this before (in production I do not deploy with docker so I've only tested the web part in docker)